### ISSUE TITLE
Moving the mouse in screen saver sometimes triggers a video player menu

### ISSUE THREAD

**Issue by IanDaBest** [rocksdanister/lively#2259]
**Describe the bug**
Sometimes, when moving the mouse in screen saver, instead of turning screen saver off, it shows a video player menu with pause, skip buttons, etc. 

**To Reproduce**
1. Trigger screen saver
2. Move your mouse
3. It will happen sometimes

**Expected behavior**
The screen saver turns off like normal

**Screenshots/Video**
![image](https://github.com/rocksdanister/lively/assets/66255263/49a70c8d-5b4f-4db0-a6d3-d5bcd511acae)

**Desktop (please complete the following information):**
 - OS: Windows Version 23H2 (OS Build 22631.3593)
- A video of Let it Go

**Log file (Important)**
[lively_log_20240602_170107.zip](https://github.com/user-attachments/files/15524427/lively_log_20240602_170107.zip)

**Comment by manstheram** [comment]
I sometimes get the ui showing up when the screensaver starts, I added this to the Mpv config in the Lively's plugin folder and it stops the video player menu from showing.
`osc=no`
However, I don't know if this is what you need as you are saying it stays stuck on the screen when you try to exit the screensaver.

**Comment by IanDaBest** [comment]
If I move the mouse, the screensaver doesn't exit but if I press other buttons (alt tab, etc.) then it exits.

**Comment by rocksdanister** [comment]
1. Are you using the latest v2.1 screensaver .scr plugin?
2. Do you have some command line script that runs screensavers?
3. Does this happen while lively running in the background or when lively is not running.

**Comment by IanDaBest** [comment]
1. How do you check that?
2. I am not using any command line scripts
3. Lively is always running in the background.

**Comment by rocksdanister** [comment]
> How do you check that?

https://github.com/rocksdanister/lively/wiki/Screen-Saver#1-windows-screensaver-file

New `Lively.scr` file is only 8KB, previous was around 25MB I think.

Screensaver is starting in preview mode but even if the osc shows briefly it should auto hide.

Like @manstheram mentioned the following `mpv.conf` file will disable ui:
```
osc=no
```
or you can try disabling the mpv mouse input with the following config:
```
input-cursor=no
```
I recommend appdata folder instead of install location:
https://github.com/rocksdanister/lively/wiki/Video-Guide#mpv

**Comment by IanDaBest** [comment]
Oh, I remember I did this a long time ago. Do I have to get the new one for the update?

**Comment by IanDaBest** [comment]
Mine is 30MB

**Comment by rocksdanister** [comment]
Yes, I get notifications when people comment so please dont comment multiple times like that.

**Comment by IanDaBest** [comment]
Sorry about that, I installed the new screensaver file and added the config file. It seems to be working fine now but we'll see.
